recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
```

Comment:
The commented # symbol is used to add comments to code, but it should not be used excessively. Consider removing it if there is no specific reason for it.